docs/proposals/20-ha-control-planes.md

```diff
@@ -528,10 +529,12 @@ In this option existing recovery mechanisms as described above are used. There i
 _Pros:_
 * Less complex to implement since no dynamic re-balancing of pods is required and there is no need to determine if there is an AZ outage.
 * Additional cost to host an HA shoot control plane is kept to the bare minimum.
-* Existing recovery mechanisms are leveraged.
+* Existing recovery mechanisms are leveraged:
+  * `kube-controller-manager` will create new replicas for the affected Deployment Pods and they will recover in short period of time.
+  * The replacement Pods for the affected StatefulSet Pods will fail to be scheduled due to their affinity rules (`etcd` Pods) or volume requirements (`prometheus` and `loki` Pods) to run in the outage zone and will not recover.
```

Comment:
I think it's quite irritating to list this a a "pro" argument, so it's rather a "con"?

Reply:
To my understanding the point here is that `Existing recovery mechanisms are leveraged` is the "pro" argument - no need to develop more sophisticated options and the existing mechanism is good enough. 

Do you suggest moving 
> * The replacement Pods for the affected StatefulSet Pods will fail to be scheduled due to their affinity rules (`etcd` Pods) or volume requirements (`prometheus` and `loki` Pods) to run in the outage zone and will not recover.

to "cons"?

I added these 2 points to clarify further what will recover (Deployment Pods) and what will not (StatefulSet Pods).

Reply:
The two points make the situation very transparent which is fine, but unfortunately saying "The replacement Pods for the affected StatefulSet Pods will fail to be scheduled due to their affinity rules ..." doesn't read as a "pro" argument anymore, does it? (Readers will probably think: "Why is that a positive thing?")
Maybe we can compensate this negative property by saying that `etcd` will still be operational because of its quorum and we can accept the downtime of logging/monitoring components for the time of an ongoing AZ outage.

Reply:
I now incorporated this feedback.